libcxx/src/support/runtime/exception_pointer_cxxabi.ipp

```diff
@@ -13,8 +13,9 @@
 
 namespace std {
 
-exception_ptr::~exception_ptr() noexcept {
-  __cxa_decrement_exception_refcount(__ptr_);
+exception_ptr::~exception_ptr() noexcept
+{
+    __cxa_decrement_exception_refcount(__ptr_);
```

Comment:
This hunk is unrelated, let's remove it.

Reply:
Done.